indra/newview/llviewerinventory.cpp

```diff
@@ -1230,6 +1230,61 @@ void create_inventory_settings(const LLUUID& agent_id, const LLUUID& session_id,
         static_cast<U8>(settype), next_owner_perm, cb);
 }
 
+void create_inventory_landmark(const LLUUID& folder_id, const std::string& name, const std::string& desc,
+    const std::string& region_name, S32 x, S32 y, S32 z)
+{
+    if (LLApp::isExiting())
+        return;
+
+    LLViewerRegion* viewer_region = gAgent.getRegion();
```

Comment:
you can use gAgent.getRegionCapability()

Reply:
Yes, but this function hides the actual problem: no region or no cap support by the region